lib/ansible/modules/cloud/ovirt/ovirt_disk.py

```diff
@@ -761,6 +763,29 @@ def main():
                     wait_condition=lambda d: d.status == otypes.DiskStatus.OK,
                     storage_domain=otypes.StorageDomain(name=module.params['image_provider']),
                 )
+        elif state == 'imported':
+            glance_service = connection.system_service().openstack_image_providers_service()
+            image_provider = search_by_name(glance_service, module.params['image_provider'])
+            images_service = glance_service.service(image_provider.id).images_service()
+            entity_id = get_id_by_name(images_service, module.params['name'])
+            images_service.service(entity_id).import_(
+                storage_domain=otypes.StorageDomain(
+                    name=module.params['storage_domain']
+                ) if module.params['storage_domain'] else None,
+                disk=otypes.Disk(
+                    name=module.params['name']
+                ),
+                import_as_template=False,
+            )
+            # Wait for disk to appear in system:
+            disk = disks_module.wait_for_import(
+                condition=lambda t: t.status == otypes.DiskStatus.OK
+            )
```

Comment:
call create, so we are sure the disk is up to date with parameters provided by user.